scripts/Makefile.am

```diff
@@ -2,3 +2,9 @@ EXTRA_DIST = wrap-compiler-for-flag-check
 if BUILD_CLIENT
   dist_bin_SCRIPTS = mosh
 endif
+
+mosh:	mosh.pl ../VERSION
+	@sed -e "s/\@VERSION\@/`cat ../VERSION`/" -e "s/\@PACKAGE_STRING\@/@PACKAGE_STRING@/" mosh.pl > mosh
```

Comment:
`$(PACKAGE_STRING)` is still better style. Automake will automatically generate the corresponding assignment `PACKAGE_STRING = @PACKAGE_STRING@`, which allows the variable to be overridden on the `make` command line if desired.